### PR TITLE
fixbug for nested name, create parent when no exists.

### PR DESCRIPTION
**Pull request by qiuzuhui**
create parent when add nested config.

### CODE REVIEW

**Comment by SBoudrias** [comment]
Can you add a test demonstrating the bug you've fixed.

**Comment by qiuzuhui** [comment]
already update the test file,   create  compress.src1 when no compress config before .

File: lib/index.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ GruntfileEditor.prototype.insertConfig = function (name, config) {
   var gruntConfig = this.gruntfile.callExpression('grunt.initConfig').arguments.at(0);
   // Select nested paths
   var targetConfig = name.split('.').reduce(function (nestedConfig, key) {
+    nestedConfig.node.range || nestedConfig.value('{}') //create empty object when value is undefined
```
Comment (SBoudrias) [inline lib/index.js]: Make this a `if` statement.

Move the comment to the line above and add a space between `//` and text.

**Comment by SBoudrias** [comment]
Also, the build is failing due to some linting errors. Could you fix these at the same time so we can merge this promptly? Thanks!

**Comment by SBoudrias** [comment]
@qiuzuhui Hey, I'd still like to fix this. It just need to pass the build. Run `npm test` locally to see the errors.

**Comment by SBoudrias** [comment]
Fixed and merged manually.

**Pull request closed without merging**